Cargo.toml

```diff
@@ -20,8 +20,9 @@ oasis-ethwasi-runtime-common = { path = "./common" }
 oasis-ethwasi-runtime-keymanager = { path = "./keymanager-runtime" }
 
 # Oasis core.
-oasis-core-runtime = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.9" }
-oasis-core-keymanager-client = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.9" }
+oasis-core-runtime = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.10" }
+oasis-core-keymanager-client = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.10" }
+ed25519-dalek = { git = "https://github.com/dalek-cryptography/ed25519-dalek", tag = "1.0.0-pre.3" }
```

Comment:
Why is this needed and wasn't needed before?

Reply:
it's not needed, just needed to be updated in cargo.lock (and this triggered that on build :-)), reverted the change